v2/api/network/customizations/load_balancer_extension.go

```diff
@@ -163,7 +164,20 @@ func fuzzySetInboundNatRule(InboundNatRule genruntime.ARMResourceSpec, embeddedI
 	if err != nil {
 		return errors.Wrap(err, "unable to check that embedded inboundNatRule is the same as inboundNatRule")
 	}
-	if string(embeddedInboundNatRuleJSON) != string(inboundNatRuleJSON) {
+
+	var embeddedInboundNatRuleJSONMap map[string]interface{}
+	err = json.Unmarshal(embeddedInboundNatRuleJSON, &embeddedInboundNatRuleJSONMap)
+	if err != nil {
+		return errors.Wrapf(err, "unable to unmarshal embeddedInboundNatRuleJSON (%s)", embeddedInboundNatRuleJSON)
+	}
+
+	var InboundNatRuleJSONMap map[string]interface{}
```

Comment:
Should be `inboundNatRuleJSONMap` (lowercase i)

Reply:
Actually so should the function parameter too.

Reply:
Done!